Dockerfile

```diff
@@ -1,15 +1,14 @@
 #
 # Build stage
 #
-FROM maven:3.9.5-eclipse-temurin-17-focal AS build
+FROM maven:3.9.6-eclipse-temurin-17-focal AS build
 COPY src /home/app/src
 COPY pom.xml /home/app
-RUN mvn -f /home/app/pom.xml clean package
+RUN mvn -f /home/app/pom.xml package
+
```

Comment:
Is it correct that there is no clean build anymore?

Reply:
well as far as I see it made no difference because we only copied the source code, I'll add it back, it does not hurt...